examples/web/index.html

```diff
@@ -501,6 +501,339 @@ <h2 class="typography-heading-medium-text">Alert</h2>
+
+          <script>
+            const HEADER_TOGGLE_SELECTOR = '[data-toggle="header"]';
+            const HEADER_BREAKPOINT = 1280;
+            const OPEN_CLASSNAME = 'is-open';
+
+            const getHeaderPaneSelectorsByToggles = () => {
+              const headerPaneSelectors = [];
+
+              document.querySelectorAll(HEADER_TOGGLE_SELECTOR).forEach((toggleEl) => {
+                const target = toggleEl.dataset.target;
+
+                if (!headerPaneSelectors.includes(target)) {
+                  headerPaneSelectors.push(target);
+                }
+              });
+
+              return headerPaneSelectors;
+            };
+
+            const getOpenHeaderPanes = () => {
+              const headerPaneSelectors = getHeaderPaneSelectorsByToggles();
+
+              if (headerPaneSelectors.length) {
+                return document.querySelectorAll(
+                  headerPaneSelectors.map((selector) => `${selector}.${OPEN_CLASSNAME}`).join(', ')
+                );
+              }
+
+              return null;
+            };
+
+            const closeOpenHeaderPanes = () => {
+              getOpenHeaderPanes().forEach((headerPaneEl) => toggleHeaderPane(headerPaneEl));
+            };
+
+            const toggleHeaderPane = (targetEl) => {
+              if (!targetEl) {
+                console.warn('👻 Boo…! Target header pane does not exist. Maybe you forgot to prefix the "data-target" selector with "#"? ');
+                return;
+              }
+
+              const headerEl = targetEl.parentElement;
+              const allHeaderToggleEls = headerEl.querySelectorAll(HEADER_TOGGLE_SELECTOR);
+              const openHeaderPaneEls = getOpenHeaderPanes();
+
+              if (targetEl.classList.contains(OPEN_CLASSNAME)) {
+                targetEl.classList.remove(OPEN_CLASSNAME);
+                allHeaderToggleEls.forEach((toggleEl) => {
+                  if (headerEl.querySelector(toggleEl.dataset.target) === targetEl) {
+                    toggleEl.setAttribute('aria-expanded', 'false');
+                  }
+                });
+
+                if (openHeaderPaneEls.length === 1) {
+                  document.removeEventListener('keydown', handleKeyPress);
+                  window.removeEventListener('resize', handleWindowResize);
+
+                  // TODO #DS-190
+                  // window.removeEventListener('click', handleClickOutside);
+                }
+              } else {
+                targetEl.classList.add(OPEN_CLASSNAME);
+                allHeaderToggleEls.forEach((toggleEl) => {
+                  if (headerEl.querySelector(toggleEl.dataset.target) === targetEl) {
+                    toggleEl.setAttribute('aria-expanded', 'true');
+                  }
+                });
+
+                if (!openHeaderPaneEls.length) {
+                  document.addEventListener('keydown', handleKeyPress);
+                  window.addEventListener('resize', handleWindowResize);
```

Comment:
I don't have a real device available, but when I emulate iPad Pro and rotate the device to portrait the menu shows up and slides away. Also when I just resize the window and make it smaller the same thing happens. I can imagine it would bother me, especially on the iPad. Can we fix this?

Reply:
I haven't found any but dirty ways to fix it.

Reply:
OK, we can try to fix it later with a clear head :)

Reply:
I filed DS-197 for this.